src/PrizePool.sol

```diff
@@ -186,6 +186,22 @@ contract PrizePool {
         return lastCompletedDrawId;
     }
 
+    function getTotalContributedBetween(uint32 _startDrawIdInclusive, uint32 _endDrawIdInclusive) external view returns (uint256) {
+        return DrawAccumulatorLib.getDisbursedBetween(totalAccumulator, _startDrawIdInclusive, _endDrawIdInclusive, alpha.intoSD59x18());
+    }
+
+    function getContributedBetween(address _vault, uint32 _startDrawIdInclusive, uint32 _endDrawIdInclusive) external view returns (uint256) {
+        return DrawAccumulatorLib.getDisbursedBetween(vaultAccumulators[_vault], _startDrawIdInclusive, _endDrawIdInclusive, alpha.intoSD59x18());
```

Comment:
I would rename it `vaultAccumulator` cause the mapping does store multiple accumulators but only return one when you query it.
```suggestion
        return DrawAccumulatorLib.getDisbursedBetween(vaultAccumulator[_vault], _startDrawIdInclusive, _endDrawIdInclusive, alpha.intoSD59x18());
```